extensions/extensions-docker/pom.xml

```diff
@@ -59,21 +85,21 @@
 
     <modules>
         <module>aissemble-quarkus</module>
-        <module>aissemble-airflow</module>
-        <module>aissemble-fastapi</module>
-        <module>aissemble-hive</module>
-        <module>aissemble-jenkins</module>
+<!--        <module>aissemble-airflow</module>-->
+<!--        <module>aissemble-fastapi</module>-->
+<!--        <module>aissemble-hive</module>-->
+<!--        <module>aissemble-jenkins</module>-->
         <module>aissemble-kafka</module>
-        <module>aissemble-metadata</module>
-        <module>aissemble-nvidia</module>
-        <module>aissemble-policy-decision-point</module>
+<!--        <module>aissemble-metadata</module>-->
+<!--        <module>aissemble-nvidia</module>-->
+<!--        <module>aissemble-policy-decision-point</module>-->
         <module>aissemble-spark</module>
         <module>aissemble-spark-infrastructure</module>
-        <module>aissemble-spark-operator</module>
+<!--        <module>aissemble-spark-operator</module>-->
         <module>aissemble-vault</module>
-        <module>aissemble-versioning</module>
-        <module>aissemble-mlflow</module>
-        <module>aissemble-model-training-api-containers</module>
+<!--        <module>aissemble-versioning</module>-->
+<!--        <module>aissemble-mlflow</module>-->
+<!--        <module>aissemble-model-training-api-containers</module>-->
```

Comment:
S: Revert this as local developers have the appropriate access keys setup to be able to build all of the images.

Reply:
Agree

- [ ]  uncomment modules